docs/framework/deployment/windows/installing-dotnet-35-windows-10.md

```diff
@@ -0,0 +1,20 @@
+# Installation the .NET Framework 3.5 on Windows 10
```

Comment:
The only other OS to call out would be Win8 since this is Win8+. I am choosing to make it Win10 specific in the title since I believe that the the Win 8.x install base is small. Win 8 is called out in the text.